### PR TITLE
[Docs](array type) Update array type documentation

### PR DESCRIPTION
**Pull request by xy720**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by adonis0147 — DISMISSED** [review]
LGTM

**Review by cambyzju — DISMISSED** [review]
LGTM

File: docs/en/docs/sql-manual/sql-reference/Data-Types/ARRAY.md [hunk 1]
```diff
@@ -40,7 +40,13 @@ DATETIME, CHAR, VARCHAR, STRING
 ```
 ### notice
 
-We should add config `enable_array_type=true` inside fe.conf before use ARRAY
+We should turn on ARRAY switch with the following command before use:
```
Comment (adonis0147) [inline docs/en/docs/sql-manual/sql-reference/Data-Types/ARRAY.md:43]:
```suggestion
We should turn on the switch for the ARRAY types feature with the following command before use:
```

**Review by adonis0147 — CHANGES_REQUESTED** [review]

File: docs/en/docs/sql-manual/sql-reference/Data-Types/ARRAY.md (hunk 1 above)
Comment (xy720) [reply docs/en/docs/sql-manual/sql-reference/Data-Types/ARRAY.md]: done

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**